AnchorageTests/AnchorageTests.swift

```diff
@@ -46,13 +46,13 @@ import XCTest
         let TestPriorityRequired = UILayoutPriority.required
         let TestPriorityHigh = UILayoutPriority.defaultHigh
         let TestPriorityLow = UILayoutPriority.defaultLow
+        public typealias ConstraintAttribute = NSLayoutConstraint.Attribute
     #else
         let TestPriorityRequired = UILayoutPriorityRequired
         let TestPriorityHigh = UILayoutPriorityDefaultHigh
         let TestPriorityLow = UILayoutPriorityDefaultLow
+        public typealias ConstraintAttribute = NSLayoutAttribute
     #endif
-
-    public typealias ConstraintAttribute = NSLayoutAttribute
```

Comment:
It looks like we mistakenly split the constraint attributes between the macOS and iOS sections. Might be nice to move them all up to the top-level `#if swift(>=4.0)` on line 18.